app/controllers/admin/news_controller.rb

```diff
@@ -0,0 +1,53 @@
+# encoding: utf-8
+class Admin::NewsController < ApplicationController
+  before_action :authorize
+
+  def index
+    @news_grid = initialize_grid(News, include: :user,
+                                       order: 'news.created_at',
+                                       order_direction: :desc)
+  end
+
+  def new
+    @news = News.new
+  end
+
+  def edit
+    @news = News.find(params[:id])
+  end
+
+  def create
+    @news = News.new(news_params)
+    @news.user = current_user
+    if @news.save!
```

Comment:
Not correct. save will return false on failure but save! will throw exception.

Try begin ... resuce instead.